prisma/schema.prisma

```diff
@@ -47,6 +47,7 @@ model User {
   accounts       Account[]
   sessions       Session[]
   documents      Document[]
+  dataroom       Dataroom[]
```

Comment:
`datarooms` would be good instead of `dataroom`

Reply:
On it